.github/workflows/testing.yml

```diff
@@ -134,7 +139,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.9
+        python-version: 3.11
```

Comment:
We might need to drop this to 3.10 since `codespell` doesn't yet work on Python 3.11, but I'm guessing it'll end up working on 3.11 sooner than Numba.